README.md

```diff
@@ -65,7 +65,7 @@ https://emscripten.org/docs/building_from_source/toolchain_what_is_needed.html.
 
 ### Linux
 
-- `python`: Version 3.9.2 or above.
+- `python`: Version 3.13.0 or above.
```

Comment:
I'm not sure we want to drop support for python 3.9 just because emsdk bundles a bleeding edge version.   I think the idea is that we still want to support the system version of python on older/stable linux releases.

Reply:
In fact I suggest we keep the linux version of emsdk pinned to the older version of python to avoid accidentally breaking compat.